File: C/Prob1.c
#include <stdio.h>
#include <time.h>

int main() {
	
	int limit = 1000;
	int sum = 0;

	clock_t time = clock();
	
	for(int i=0;i<limit;i++){
		if(i%3==0 || i%5==0){
			sum += i;
		}
	}
	
	time = clock()-time;

	printf("<C> Sum of digits less than %d: %d  (found in %f seconds)\n", limit, sum, ((double)time/CLOCKS_PER_SEC));

	return 0;
}

File: python/Prob1/Prob1.py
#!/usr/bin/python

import time

i = 1000
sum = 0

start = time.time()

for x in range(0,i):
	if(x%3==0 or x%5==0):
		sum = sum + x

end = time.time()

print("<Python> Sum of digits less than %d: %d  (found in %s)" % (i,sum,str(end-start)))
